src/enzo/Star_ComputePhotonRates.C

```diff
@@ -30,19 +30,22 @@
 #include "LevelHierarchy.h"
 
 float ReturnValuesFromSpectrumTable(float ColumnDensity, float dColumnDensity, int mode);
+float CalculateRotationalPhotonRates(float Mass, float age, int species);
+float CalculateNonRotationalPhotonRates(float Mass, float age, int species);
 
 int Star::ComputePhotonRates(const float TimeUnits, int &nbins, float E[], double Q[])
 {
 
 
   int i;
-  double L_UV, cgs_convert, _mass;
+  double L_UV, cgs_convert, _mass, _age;
   float x, x2, EnergyFractionLW, MeanEnergy, XrayLuminosityFraction;
   float Mform, EnergyFractionHeI, EnergyFractionHeII;
   if (this->Mass < 0.1)  // Not "born" yet
     _mass = this->FinalMass;
   else
     _mass = this->Mass;
+    _age = this->ReturnAge()/this->LifeTime; //Percentage of lifetime
```

Comment:
The indentation here makes this looks like _age is part of the else block, but there are no brackets ({}), so it is not. Maybe fix indentation to make it clear?